packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -188,6 +192,45 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  'DropdownButton': (BuildContext context, DataSource source) {
+    final length = source.length(['items']);
+    final dropdownMenuItems = List<DropdownMenuItem<Object>>.generate(
+        length,
+        (i) => DropdownMenuItem<Object>(
+            onTap: source.voidHandler(['items', i, 'onTap']),
+            value: source.v<String>(['items', i, 'value']),
```

Comment:
does it matter that this has to be a string, given that the "current" value can be a string, int, double, or bool?

Reply:
You're right. This can be any given type. Fixed.